orienteer-core/src/main/resources/org/orienteer/core/web/orienteer.js

```diff
@@ -1,2 +1,91 @@
 'use strict';
 
+(function () {
```

Comment:
Why this script is needed? Can OOB CoreUI handle that?

Reply:
This script need for handle Orienteer UI for different screen sizes.